### ISSUE TITLE
Startup time too high

### ISSUE THREAD

**Issue by gilcu3** [JurajNyiri/HomeAssistant-Tapo-Control#543]
### Description

Recently homeassistant added a way to measure startup time per integration. After checking that, it seems that this integration takes 11s to start in my environment. I have only two cameras, C100 and C200.

### Reproduction Steps

1. Start homeassistant

### Expected behavior

Shouldn't it be much faster?

### If applicable, add error logs.

```
Apr 04 17:18:39 rpi5 hass[41830]: 2024-04-04 17:18:39.081 WARNING (MainThread) [homeassistant.components.light] None (<class 'custom_components.tapo_control.light.TapoWhitelight'>) does not set supported color modes, this will stop working in Home Assistant Core 2025.3, please create a bug report at https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues
Apr 04 17:18:51 rpi5 hass[41830]: 2024-04-04 17:18:51.452 WARNING (MainThread) [homeassistant.components.select] Setup of select platform tapo_control is taking over 10 seconds.
```

### Device Firmware

1.3.11 Build 231115 Rel.85400n(4555)

### Integration Version

5.4.17

### Using stream component

Yes

### Does camera work via official integrations?

Yes

### Camera has all attributes filled out in developer tools

Yes

### HASS Environment

core

### Search for similar issues

Yes

**Comment by JurajNyiri** [comment]
Startup time depends on camera and also onvif communication initiation.

**Comment by gilcu3** [comment]
> Startup time depends on camera and also onvif communication initiation.

I have both connected to the same local network. Does your answer mean that this is kinda normal behavior, or is there anything to do to improve it?

**Comment by JurajNyiri** [comment]
Yes it is normal behaviour. There might be space to refactor and improve but that is very low priority at this point.
